src/main/java/core/basesyntax/strategy/impl/OperationStrategyImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.OperationStrategy;
+import java.util.HashMap;
+import java.util.Map;
+
+public class OperationStrategyImpl implements OperationStrategy {
+    private Map<FruitTransaction.Operation, OperationHandler> operationHandlerMap = new HashMap<>();
+
+    @Override
+    public Map<FruitTransaction.Operation, OperationHandler> createOperationHandlerMap() {
+        operationHandlerMap.put(FruitTransaction.Operation.BALANCE, new BalanceHandler());
```

Comment:
better pass the whole map to the constructor as a dependency, this will make this class more flexible